diff --git a/include/core/SkPostConfig.h b/include/core/SkPostConfig.h
index 6e54bfd..e672f35 100644
--- a/include/core/SkPostConfig.h
+++ b/include/core/SkPostConfig.h
@@ -385,27 +385,27 @@
 
 #ifndef SK_ATOMICS_PLATFORM_H
 #  if defined(_MSC_VER)
-#    define SK_ATOMICS_PLATFORM_H "../../src/ports/SkAtomics_win.h"
+#    define SK_ATOMICS_PLATFORM_H "../ports/SkAtomics_win.h"
 #  else
-#    define SK_ATOMICS_PLATFORM_H "../../src/ports/SkAtomics_sync.h"
+#    define SK_ATOMICS_PLATFORM_H "../ports/SkAtomics_sync.h"
 #  endif
 #endif
 
 #ifndef SK_MUTEX_PLATFORM_H
 #  if defined(SK_BUILD_FOR_WIN)
-#    define SK_MUTEX_PLATFORM_H "../../src/ports/SkMutex_win.h"
+#    define SK_MUTEX_PLATFORM_H "../ports/SkMutex_win.h"
 #  else
-#    define SK_MUTEX_PLATFORM_H "../../src/ports/SkMutex_pthread.h"
+#    define SK_MUTEX_PLATFORM_H "../ports/SkMutex_pthread.h"
 #  endif
 #endif
 
 #ifndef SK_BARRIERS_PLATFORM_H
 #  if SK_HAS_COMPILER_FEATURE(thread_sanitizer)
-#    define SK_BARRIERS_PLATFORM_H "../../src/ports/SkBarriers_tsan.h"
+#    define SK_BARRIERS_PLATFORM_H "../ports/SkBarriers_tsan.h"
 #  elif defined(SK_CPU_ARM32) || defined(SK_CPU_ARM64)
-#    define SK_BARRIERS_PLATFORM_H "../../src/ports/SkBarriers_arm.h"
+#    define SK_BARRIERS_PLATFORM_H "../ports/SkBarriers_arm.h"
 #  else
-#    define SK_BARRIERS_PLATFORM_H "../../src/ports/SkBarriers_x86.h"
+#    define SK_BARRIERS_PLATFORM_H "../ports/SkBarriers_x86.h"
 #  endif
 #endif
 
